src/elements/Img-modal.vue

```
@@ -1,67 +1,55 @@
-    },
-  }
+  const props = defineProps<{ image: ImageField }>()
+
+  const thumbnailField = (props.image as ImageField<'thumbnail'>).thumbnail ?? props.image
+
+  const fullWidthImage = ref(false)
 </script>
 
+<template>
+  <button class="thumbnail-btn" @click="fullWidthImage = true">
+    <PrismicImage :field="thumbnailField" width="200" height="300" loading="lazy" />
```

ESLint

[error] Replace `·:field="thumbnailField"·width="200"·height="300"·loading="lazy"` with `⏎······:field="thumbnailField"⏎······width="200"⏎······height="300"⏎······loading="lazy"⏎···`

(prettier/prettier)